.github/workflows/release_draft.yml

```diff
@@ -55,11 +55,14 @@ jobs:
               repo: context.repo.repo,
               ref: 'tags/'
             });
-            console.log(`Existing tags: ${refs.map(ref => ref.ref.replace('refs/tags/', '')).join(', ')}`);
 
-            const latestTag = refs.sort((a, b) => new Date(b.object.date) - new Date(a.object.date))[0].ref.replace('refs/tags/', '');
-            console.log(`Latest tag: ${latestTag}`);
+            if (refs.length === 0) {
+              // No existing tags, so any new tag is valid
+              console.log('No existing tags found. Any new tag is considered valid.');
+              return;
+            }
 
+            const latestTag = refs.sort((a, b) => new Date(b.object.date) - new Date(a.object.date))[0].ref.replace('refs/tags/', '');
```

Comment:
This is wrong. Don't sort by date but by version expression. (if you are changing it)